internal/controllers/device_data_controller_test.go

```diff
@@ -351,9 +358,12 @@ func TestUserDevicesController_QueryDeviceErrorCodes(t *testing.T) {
 		response, _ := app.Test(request)
 		body, _ := io.ReadAll(response.Body)
 
+		chtJSON, err := json.Marshal(chatGptResp)
+		assert.NoError(t, err)
+
 		assert.Equal(t, fiber.StatusOK, response.StatusCode)
 		assert.Equal(t,
-			fmt.Sprintf(`{"message":"%s"}`, chatGptResp),
+			fmt.Sprintf(`{"clearedAt":"%s","errorCodes":%s}`, null.Time{}.Time.UTC().Format(time.RFC3339Nano), chtJSON),
```

Comment:
I don't think having the zero time appear here is helpful.

I'm going to change this.